package.json

```diff
@@ -61,6 +61,7 @@
 		"path-key": "^4.0.0",
 		"tempfile": "^4.0.0",
 		"tsd": "^0.19.1",
+		"type-fest": "^3.5.7",
```

Comment:
I believe this should be a production dependency? Otherwise, TypeScript users that do `npm install execa` would receive a type error due to the missing dependency.

`type-fest` is rather big, so I would be wondering how @sindresorhus feels about introducing that dependency :thinking:

Reply:
I have since removed `type-fest` with the refactor to use type overloads: https://github.com/sindresorhus/execa/pull/510/commits/6a283cb0dd156c4060b3935f253be758dfceba98